doc/whats_new/v1.0.rst

```diff
@@ -180,6 +180,13 @@ Changelog
   `Thomas Fan`_ and :user:`Amanda Dsouza <amy12xx>` and
   :user:`EL-ATEIF Sara <elateifsara>`.
 
+:mod:`sklearn.inspection`
```

Comment:
Could you merge with master and move this section to respect the alphabetic order